core/src/main/java/io/kestra/core/runners/pebble/expression/InExpression.java

```diff
@@ -0,0 +1,42 @@
+package io.kestra.core.runners.pebble.expression;
+
+import io.pebbletemplates.pebble.node.expression.BinaryExpression;
+import io.pebbletemplates.pebble.node.expression.Expression;
+import io.pebbletemplates.pebble.template.PebbleTemplateImpl;
+import io.pebbletemplates.pebble.template.EvaluationContextImpl;
+
+import java.util.Collection;
+import java.util.Objects;
+
+public class InExpression extends BinaryExpression<Object> {
+    public InExpression() {
+    }
+
+    public InExpression(Expression<?> left, Expression<?> right) {
+        super(left, right);
+    }
+
+    @Override
+    public Object evaluate(PebbleTemplateImpl self, EvaluationContextImpl context) {
+        Object leftValue = getLeftExpression().evaluate(self, context);
+        Object rightValue = getRightExpression().evaluate(self, context);
+
+        if (rightValue instanceof Collection<?>) {
+            return ((Collection<?>) rightValue).contains(leftValue.toString());
```

Comment:
I have some doubt here, if you didn't  map all collection item with `toString()` the contains will only work for strings right?